src/apps/vpn/serverlatency.cpp

```diff
@@ -52,6 +54,15 @@ void ServerLatency::initialize() {
   connect(&m_pingTimeout, &QTimer::timeout, this,
           &ServerLatency::maybeSendPings);
 
+  connect(tutorial, &Tutorial::playingChanged, this, [this, tutorial](){
+      if(tutorial->isPlaying()) {
+          stop();
+          m_refreshTimer.stop();
```

Comment:
@oskirby Decided to stop the timer since QTimer does not have built in pausing functionality, and with the refresh timer being frequent at 30 minutes, I feel it is okay to restart the timer here. Let me know what you think